client/hooks/useStyles.js

```diff
@@ -0,0 +1,337 @@
+import useTheme from './useTheme';
```

Comment:
This needs to be more dynamic. I was thinking we can pass in object with styles and get back stylesheet version. Needs to account for currenttheme. Otherwise this file is just going to become even larger

Reply:
Open to other solutions though

Reply:
Yh because of the size I stopped to actually ask the question yes using an object would be a better idea but for now won’t be able to merge or get rid of similar styles.